tests/integration/helpers/network.py

```diff
@@ -59,16 +111,36 @@ def create_rule(src, dst):
                 rule["destination_port"] = port
             return rule
 
+        def create_rule_v6(src, dst):
+            rule = {
+                "source": src.ipv6_address,
+                "destination": dst.ipv6_address,
+                "action": action,
+            }
+            if port is not None:
+                rule["destination_port"] = port
+            return rule
+
         self._add_rule(create_rule(left, right))
         self._add_rule(create_rule(right, left))
 
+        if left.ipv6_address and right.ipv6_address:
```

Comment:
The upper lines are without conditions. Does it mean we always want to create ipv4 unconditionally?

Reply:
Right.